.changelog/1432.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+pages: Added mTLS and Vectorize bindings to pages_project
```

Comment:
Missing AE, Queue Producers and AI bindings

Reply:
They're coming :)